### DISCUSSION TITLE
Moon Single-Purchase per run "Upgrade"/mode

### DISCUSSION THREAD

**Discussion by Skyblade799** [Malcolm-Q/LC-LateGameUpgrades#521] [Ideas]
I couldn't find a mod for this, and then it occurred to me that LGU might be a good fit for something like this.
Basically, I wish there was a toggle feature in the config that would let the moons themselves be "upgrades", where you would purchase the moon routing cost, and it would be set to zero in the future (effectively making it free for the rest of that run, and reset once fired). 

It could probably be an alternate option to the upgrade that reduces routing costs in the config (as that upgrade probably wouldn't make much sense with this), and could come with a multiplier to scale the price of moons.

**Comment by WhiteSpike** [comment]
Does [Lethal Moon Unlocks](https://thunderstore.io/c/lethal-company/p/explodingMods/LethalMoonUnlocks/) not work for what you described?

**Comment by Skyblade799** [comment]
Sorry for the late reply, but yes, that mod should work! Didn't know it existed.
